tests/test_flask_application.py

```diff
@@ -220,31 +222,60 @@ async def test_admit(
     out,
     exception,
 ):
+    session = aiohttp.ClientSession()
     with exception:
         if index == 8:
             monkeypatch.setenv("AUTOMATIC_UNCHANGED_APPROVAL", "1")
         with aioresponses() as aio:
             aio.get(re.compile(r".*"), callback=fix.async_callback, repeat=True)
-            response = await pytest.fa.__admit(AdmissionRequest(adm_req_samples[index]))
+            response = await pytest.fa.__admit(
+                AdmissionRequest(adm_req_samples[index]), session
+            )
             assert response == out
 
 
 @pytest.mark.parametrize(
-    "function, err",
+    "function, err, status",
     [
         (
             {
                 "target": "connaisseur.flask_application.send_alerts",
                 "side_effect": exc.AlertSendingError(""),
             },
             "Alert could not be sent. Check the logs for more details!",
+            500,
```

Comment:
Currently, if alerting fails and Connaisseur is configured to require alerts to be sent, Connaisseur fails. We could try to catch this and respond with an error that explains, we deny the image because of failed alert. Thoughts?

Reply:
would consider that to be important if it happens

Reply:
Ok, will open an issue